lms/services/course.py

```diff
@@ -70,6 +70,30 @@ def upsert_course(self, context_id, name, extra, settings=None) -> Course:
             type_=Grouping.Type.COURSE,
         )[0]
 
+    def find_group_set(self, group_set_id=None, name=None, context_id=None):
+        from sqlalchemy import func, column, Text
+
+        group_set = (
+            func.jsonb_to_recordset(Course.extra["group_sets"])
```

Comment:
`recordset` is basically the same as `rows` so this will get every element of a list and present them as rows.